### PR TITLE
Remove whitespace from empty lines in the test classes

### PR DESCRIPTION
**Pull request by mikelupo**
### Checklist

- [x] I've checked that all new and existing tests pass
- [x] I've updated the documentation if necessary
- [X] I've added an entry in the CHANGELOG to credit myself

### Description

Remove unnecessary whitespace from empty lines in the test classes.

### Motivation and Context

Future merge issues. Adheres to the already high standard of OHHTTPStubs

Ran all of the tests. 100% passing.

### CODE REVIEW

**Comment by AliSoftware** [comment]
Thx!
Question: why only do it on test classes and not the whole codebase?

**Comment by mikelupo** [comment]
Time. Leave this open and I'll get to it this week 🙏 ??

**Comment by AliSoftware** [comment]
Haha sure, no rush! 👍

**Comment by mikelupo** [comment]
ok, I got to it. 🧀 😁

File: OHHTTPStubs/Pods/AFNetworking/AFNetworking/AFNetworkReachabilityManager.h [hunk 1]
```diff
@@ -75,7 +75,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  Creates and returns a network reachability manager with the default socket address.
- 
+
```
Comment (AliSoftware) [inline OHHTTPStubs/Pods/AFNetworking/AFNetworking/AFNetworkReachabilityManager.h]: Careful, we shouldn't modify the code of third-party pods 😉

Maybe re-run `pod install` after your regex to overwrite the pods's code back to their original code for example

Comment (mikelupo) [inline OHHTTPStubs/Pods/AFNetworking/AFNetworking/AFNetworkReachabilityManager.h]: That's what I get for trying to do this fast. 👎

**Comment by mikelupo** [comment]
Instead of adding a new commit, I amended the last one. I hope that is OK with the github workflow.

**Comment by AliSoftware** [comment]
Perfect 👍

**Pull request merged**